cli/tauri-bundler/src/bundle/osx_bundle.rs

```diff
@@ -76,9 +76,25 @@ pub fn bundle_project(settings: &Settings) -> crate::Result<Vec<PathBuf>> {
 
   create_path_hook(&bundle_directory, settings).chain_err(|| "Failed to create _boot wrapper")?;
 
+  if settings.osx_signing_identity().is_some() {
+    sign(app_bundle_path.clone(), settings)?;
+  }
+
   Ok(vec![app_bundle_path])
 }
 
+fn sign(app_bundle_path: PathBuf, settings: &Settings) -> crate::Result<()> {
+  let identity = settings.osx_signing_identity().expect("signing identity not provided");
+  common::print_info(format!(r#"signing app with identity "{}""#, identity).as_str())?;
+  let status = Command::new("codesign")
+    .args(vec!["--deep", "--force", "-s", identity, &app_bundle_path.to_string_lossy()])
+    .status()?;
+  if !status.success() {
+    return Err(crate::Error::from("failed to sign app"));
```

Comment:
Can we show the `stderr` output in case signing fails? That should make it easier for users to figure out what's going wrong

Reply:
yeah that makes sense, gonna do it